src/CarbonAware.WebApi/test/unitTests/Filters/CarbonAwareParametersBaseDtoSchemaFilterTests.cs

```diff
@@ -0,0 +1,49 @@
+using CarbonAware.WebApi.Filters;
+using Microsoft.OpenApi.Models;
+using NUnit.Framework;
+using Swashbuckle.AspNetCore.SwaggerGen;
+using GSF.CarbonAware.Handlers.CarbonAware;
+using System.Text.Json;
+
+namespace CarbonAware.WepApi.UnitTests;
+
+[TestFixture]
+
+public class CarbonAwareParametersBaseDtoSchemaFilterTests
```

Comment:
Maybe we can take this offline and you elaborate more,  since I'm not sure what the utest is about.

Reply:
Will ping you to discuss!